### DISCUSSION TITLE
UpdatesInterval Specify Day

### DISCUSSION THREAD

**Discussion by zeehaque** [Romanitho/Winget-AutoUpdate#355] [Q&A] [answered]
Hi all,

I apologize in advance for what may be a very simple question - can the **-UpdatesInterval** parameter be used to specify a certain day of the week?

For instance, if we would like to run updates every Friday, is -UpdatesInterval Weekly Friday or something similar possible?

In terms of the -UpdatesInterval _Weekly_ setting, would it run weekly based on the day WAU was installed with that setting on the device?

Thanks!

**Comment by Romanitho** [comment, marked as answer]
Hi,
By default, WAU will run on tuesday. You can adjuste this inside the install script before running it. Have a look in the `Winget-AutoUpdate-Install.ps1` from line 288 and change the day of week with the one you want:

```
elseif ($UpdatesInterval -eq "Weekly") {
    $tasktriggers += New-ScheduledTaskTrigger -Weekly -At $UpdatesAtTime -DaysOfWeek 2
}
elseif ($UpdatesInterval -eq "BiWeekly") {
    $tasktriggers += New-ScheduledTaskTrigger -Weekly -At $UpdatesAtTime -DaysOfWeek 2 -WeeksInterval 2
}
elseif ($UpdatesInterval -eq "Monthly") {
    $tasktriggers += New-ScheduledTaskTrigger -Weekly -At $UpdatesAtTime -DaysOfWeek 2 -WeeksInterval 4
}
```

**Reply by zeehaque** [reply]
Thanks!! That did it.

**Reply by Romanitho** [reply]
;)

**Reply by nortonrome** [reply]
Is there any way to change the week day afterwards? We provide WAU via MS Store and use the Winterlights fork to configure via intune.

Thanks

**Reply by Romanitho** [reply]
Don't use the fork; we don't maintain it here. We provide everything here to run WAU through Intune.

**Reply by nortonrome** [reply]
Ok that was just an additional information. I am still curious if it's possible to change the specific day afterwards after having installed it to several devices already?

**Reply by Romanitho** [reply]
Still work progress:
https://github.com/Romanitho/Winget-AutoUpdate/pull/839
